config/locales/sufia.en.yml

```diff
@@ -2,7 +2,7 @@ en:
   curation_concerns:
     institution_name:   "CHF"
   sufia:
-    product_name:       "CHF Digital Collections (Beta)"
+    product_name:       "Digital Collections"
```

Comment:
Looks like there's a helper that overrides application_name with the value of product_name.